torch/_inductor/ops_handler.py

```diff
@@ -744,6 +756,10 @@ def frexp(x) -> Tuple[None, None]:
     def scan(dtypes, combine_fn, values) -> Tuple[None, ...]:
         return tuple(None for i in range(len(values)))
 
+    @staticmethod
+    def sort(dtypes, values, stable, descending) -> Tuple[None, ...]:
+        return tuple(None for i in range(len(values)))
```

Comment:
nit. and it follows the pattern above, but we can simply do `(None,) * len(values)`